Kitodo/src/test/resources/mapping.json

```diff
@@ -0,0 +1,648 @@
+{
```

Comment:
Same indent mix and did we need really to JSON files? I think one should be used for real and test usage as both will differ in future and tests did not check the real execution.

Reply:
and how to enforce that test execution uses mapping from outside?

Reply:
So far as I could discover it is not so easy to solve this or I did not found a good source for knowledge. We must stay with both files until we found a good and practicable way and changing of both files must be done.